src/oci-umount.c

```diff
@@ -391,7 +395,13 @@ static int unmount(const char *id, char *umount_path, bool submounts_only, const
 		if (mnt_table[i].parent_mntid != mntid)
 			continue;
 		ret = umount2(mnt_table[i].destination, MNT_DETACH);
-		if (!ret)
+		if (!ret) {
+			if (errno == EINVAL)
+				pr_pdebug("Not a mount point [%s]: %m", umount_path);
```

Comment:
I am still not sure that why this change is required. I think existing code already takes care of it. Before we call umount, we check if this is a mount point or not. And for submounts, we get list from /proc/self/mountinfo, so it has to be a mountpoint. So this does not seem like an issue at all?